Autonomous_Parallel_Parking/stm32f3discovery.cfg

```diff
@@ -1,9 +1,10 @@
 # This is an STM32F3 discovery board with a single STM32F303VCT6 chip.
 # http://www.st.com/internet/evalboard/product/254044.jsp
 
-source [find interface/stlink-v2-1.cfg]
+source [find interface/stlink-v2.cfg]
```

Comment:
This was what I had to change to help load the firmware onto the lab boards. In addition have to remove left u sensor for loading firmware